---
sinks:
  - type: ReconstructionVisualizer
    ns: ~reconstruction
    min_weight: 0.0
    max_weight: 1000.0
    min_distance: -0.3
    max_distance: 0.3
    slice_height: 0.0
    use_relative_height: true
    min_hue: 0.7
    max_hue: 0.9
    min_saturation: 0.0
    max_saturation: 0.9
    min_luminance: 0.0
    max_luminance: 0.85
  - type: TsdfOccupancyPublisher
    extraction:
      min_distance: 0.25
